lib/node_modules/@stdlib/stats/base/nanvariancetk/test/test.main.js

```diff
@@ -148,6 +202,60 @@ tape( 'the function calculates the sample variance of a strided array (ignoring
 	t.end();
 });
 
+tape( 'the function calculates the sample variance of a strided array (ignoring `NaN` values) (accessors)', function test( t ) {
```

Comment:
```suggestion
tape( 'the function calculates the sample variance of a strided array (accessors)', function test( t ) {
```
kindly update these test messages everywhere else in this PR